gen/nvml/nvml.h.patch

```diff
@@ -1,56 +1,61 @@
---- nvml.h	2020-09-24 12:43:36.060860551 +0000
-+++ nvml.patched.h	2020-09-24 12:45:41.412702233 +0000
-@@ -128,7 +128,9 @@
+--- gen/nvml/nvml.h	2021-01-14 17:28:34.075357700 +0100
++++ -	2021-01-14 17:28:49.972647500 +0100
+@@ -131,7 +131,10 @@
   */
  #define NVML_VALUE_NOT_AVAILABLE (-1)
  
 -typedef struct nvmlDevice_st* nvmlDevice_t;
-+typedef struct {
-+	struct nvmlDevice_st* handle;
++typedef struct
++{
++    struct nvmlDevice_st* handle;
 +} nvmlDevice_t;
```

Comment:
Why did this change? Looks like maybe it was updated to adhere to the consistent format for struct in the rest of this file (which is a good change, actually if that's the reason).

Ditto to similar ones below.

Reply:
When looking at the input file, the convention used seems to be to start the block on the line following the typedef. For this reason, I constructed the `sed` command to generate the `patch` file to also break after the `typedef` line to ensure uniformity (as you assumed).

From the README:
```bash
sed -E 's#(typedef\s+struct)\s+(nvml.*_st\*)\s+(nvml.*_t);#\1\n{\n    struct \2 handle;\n} \3;#g' gen/nvml/nvml.h \
    | diff -u gen/nvml/nvml.h - > gen/nvml/nvml.h.patch
```

Reply:
As a note, I have added a make target that updates the file instead of using a patch file. We can revisit if there are additional changes that need to be made to the file that are not purely structural.